src/control/cmd/dmg/pretty/pool_test.go

```diff
@@ -432,7 +432,7 @@ Target: type unknown, state up_in
 - Storage tier 1 (NVMe):
   Total size: 100 GB
   Free: 90 GB
-- Storage tier 2 (unknown):
+- Storage tier 2 (QLC):
```

Comment:
We should add another tier in these tests to guarantee that unknown tiers print correctly.

Reply:
ok, will do it.